.travis.yml

```diff
@@ -11,7 +11,10 @@ cache:
   directories:
     - $HOME/.cache # cython cache
     - $HOME/.ccache # compiler cache
-
+addons:
+  apt:
+    packages:
```

Comment:
this has not worked welll in travis - pls remove

Reply:
@jreback ,

Thanks for the feedback. I have added it to install xvfb. It is getting successfully installed in Travis check. May I know where can I add the apt-get command to install it.

Reply:
If we don't need it for the other builds, we shouldn't need this to this PR. Not sure why you added it, but this is required only for the clipboard tests, which you are explicitly not running in the new build.